src/main/java/com/treasure/hunt/strategy/geom/GeometryItem.java

```diff
@@ -16,10 +16,4 @@
 public class GeometryItem<T extends Geometry> {
     @NonNull T object;
     GeometryType type;
```

Comment:
Init with standard GeometryType would be nice

Reply:
What do you mean?